README.md

```diff
@@ -34,20 +38,18 @@ https://github.com/udacity/CarND-Capstone/issues/147 We locally updated our requ
 * Real world testing (will be updated)
 See below
 
-* Suggestions
-I wish Virtual Workspace had internet access so that instead of uploading files it would be possible to clone a repo.
 
 ### Other Issues we ran into
 1. Requirements.txt was modified to point pillow to version 4.3 (see above).  We had an issue with cv.Bridge that was traced back to https://github.com/udacity/CarND-Capstone/issues/147.  This required upgrade to pillow version 4.3
```

Comment:
I have already replaced 4.3 with 2.2.1  - works fine when natively installed. Let's *rephrase* this point - that for Docker installation it would be needed to have 4.3, but Workspace & native installation were just fine.

Reply:
leave reference to the issue as is - it will show that we have spent some time on the problem.